transport/grpc/doc.go

```diff
@@ -40,6 +40,30 @@
 //     Inbounds: yarpc.Inbounds{myInbound},
 //   })
 //
+// To configure TLS on your service listener, pass
+// credentials.TransportCredentials as an InboundCredentials InboundOption.
+// There are various ways to create credentials.TransportCredentials. See
+// https://godoc.org/google.golang.org/grpc/credentials#TransportCredentials.
+//
+//   listener, err := net.Listen("tcp", ":4443")
+//   if err != nil {
+//     return err
+//   }
+//
+//   myTLSConfig := &tls.Config{
+//     // any arbitrary valid tls.Config
+//   }
+//   myTransportCredentials := credentials.NewTLS(myTLSConfig)
+//   myInboundCredentials := InboundCredentials(myTransportCredentials)
+//   myInbound := grpcTransport.NewInbound(
+//     listener,
+//     []InboundOption{myInboundCredentials},
```

Comment:
This is a variadic argument so you wouldn't need to create the slice.

    grpcTransport.NewInbound(listener, InboundCredentials(myTransportCredentials))

Reply:
done